src/Worlds/DebugWorld/DebugWorld_LoggingChain.layer

```diff
@@ -0,0 +1,17 @@
+GenericEntity : "{BC5061EBB3284158}Prefabs/Weapons/Axe.et" {
+ components {
+  ActionsManagerComponent "{A29AE67FF4D82B0F}" {
+   ActionContexts {
+    UserActionContext "{5086F9ADF588DCA4}" {
+     Position PointInfo "{5086F9ADE97650DF}" {
+      Offset 0 0 0
+     }
+    }
+   }
+  }
+ }
```

Comment:
This override should not be needed. Check why the point info at 0 0 0 miss-matches the base prefab, please.

Reply:
I think this is resolved?